Sources/ProjectDescription/TestAction.swift

```diff
@@ -101,4 +130,42 @@ public struct TestAction: Equatable, Codable {
                   language: language,
                   region: region)
     }
+
+    public static func testPlans(default: Path,
```

Comment:
I like that. will do.

Reply:
I tried, but it would require me to add two more static functions because we need to support both `config: PresetBuildConfiguration` and `configurationName: String`.

It's already quite redundant, so maybe it's better to refactor configuration handling a bit first to get rid of that redundancy?